src/react-queries/index.ts

```diff
@@ -80,7 +80,11 @@ const common = createQueryKeys('common', {
       const maybeIsTradingActive = await isTradingActive(assetId, swapperName)
 
       // Do not return things in a monadic way so that we can leverage native react-query error-handling
-      if (maybeIsTradingActive.isErr()) throw maybeIsTradingActive.unwrapErr()
+      if (maybeIsTradingActive.isErr()) {
+        console.warn(maybeIsTradingActive.unwrapErr())
+        return false
```

Comment:
if we failed to get `isTradingActive` assume it is not instead of throwing. do we care about the console.warn?

waiting on @gomesalexandre to validate this is sane. otherwise, we will need to update `swapperApi` to gracefully handle the throw accordingly

Reply:
- Since this is expected flow, no need for console.warning IMO
- Indeed, we probably don't want to throw since not having an inbound address *is* expected for long-tails

Ideally, we probably still should throw when `isTradingActive` *is* an actual error, i.e long-tails would simply return the trading active status of their matching pool (`ETH.ETH` currently) and throw if an actual error happened, but that's not the current flow and is definitely for a follow-up, let's keep it like this and keep the comment above as well.

The reason for letting things throw (in legit cases) is so that the error would *not* be cached and the next request would be guaranteed fresh, though that's more paranoia than anything else since we *should* have fresh data on mount, args change, window focus etc, but modals can be derp-y sometimes and stay technically mounted while visually hidden

Reply:
ah good context on the caching bit and sounds good, leaving as is for now minus the log and we can follow up with additional changes. thanks!

Reply:
removed error logging: https://github.com/shapeshift/web/pull/6196/commits/6faa8486e2dd1642d98687d68061437fda4dfd59